clustering/common/src/main/java/org/jboss/as/clustering/controller/PropertiesAttributeDefinition.java

```diff
@@ -132,7 +132,7 @@ public Builder(PropertiesAttributeDefinition basis) {
         @Override
         public PropertiesAttributeDefinition build() {
             if (this.elementValidator == null) {
-                this.elementValidator = new ModelTypeValidator(ModelType.STRING, this.isAllowNull(), this.isAllowExpression());
+                this.elementValidator = new ModelTypeValidator(ModelType.STRING, this.isNillable(), this.isAllowExpression());
```

Comment:
isNillable() is not a method of MapAttributeDefinition.Builder.

Reply:
OK - this compiles now.  I did not realize that this PR was dependent on a future wildfly-core release.